.cirrus.yml

```diff
@@ -51,14 +51,15 @@ style_task:
     - cmake --build build --target package
   upload_script: >
     if [ "$CIRRUS_BRANCH" = "master" ]; then
-      scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i upload_key build/$(cat PACKAGE_NAME).tar.gz mule@tenzir.dfn-cert.de:artifacts/
+      rsync --archive --compress --rsh "ssh -i upload_key -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" \
+        --progress build/$(cat PACKAGE_NAME).tar.gz mule@tenzir.dfn-cert.de:/home/mule/artifacts/
```

Comment:
```suggestion
        --progress "build/$(cat PACKAGE_NAME).tar.gz" mule@tenzir.dfn-cert.de:/home/mule/artifacts/
```

Consider quoting this to avoid an error if the result of `cat PACKAGE_NAME` contains a space—it currently does if a tag contains a space, and we don't want to rely on that. The same reasoning applies above as well.

I recommend installing [shellcheck](https://www.shellcheck.net) so this gets caught automatically by a linter.